### PR TITLE
CYS - Update the full composability layout to match designs

### PR DESCRIPTION
**Pull request by albarin**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to `wp-admin/tools.php?page=woocommerce-admin-test-helper`, click the `Features` tab, and enable the `pattern-toolkit-full-composability` feature flag.
2. Go to `wp-admin/admin.php?page=wc-admin&path=%2Fcustomize-store` and click `Customize your theme` or `Start designing`.
3. Wait for the assembler to load and click on `Design your homepage`.
4. Check the layout style matches the design:

| Before | After |
|--------|--------|
| Screenshot 2024-07-10 at 09 49 46 | Screenshot 2024-07-10 at 09 50 23 |

### Changelog entry

-   [x] Automatically create a changelog entry from the details below.

-   [ ] This Pull Request does not require a changelog entry. (Comment required below)

<details>

<summary>Changelog Entry Details</summary>

#### Significance

-   [ ] Patch
-   [x] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [x] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Message 
CYS - Update the full composability layout styles

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>

### CODE REVIEW

File: plugins/woocommerce-admin/client/customize-store/assembler-hub/style.scss [hunk 1]
```diff
@@ -587,7 +586,7 @@ body.woocommerce-assembler {
 
 	.customize-your-store-edit-site-resizable-frame__inner-content {
 		height: 100%;
-		border-radius: 20px !important;
+		border-radius: 12px !important;
```
Comment (danielwrobert) [inline plugins/woocommerce-admin/client/customize-store/assembler-hub/style.scss]: I know the `!important` bit isn't an addition from this PR but I see a few of these declarations throughout this file. Is there a way to add a little more specificity and avoid using `!important`?

**Review by danielwrobert — APPROVED** [review]
LGTM! 

I left a comment about the `!important` declarations. As noted there, I realize those are not specifically related to this PR and it's not a blocker.

Aside from that question, everything tests as described! 🚀

File: plugins/woocommerce-admin/client/customize-store/assembler-hub/style.scss (hunk 1 above)
Comment (albarin) [reply plugins/woocommerce-admin/client/customize-store/assembler-hub/style.scss]: Good catch, it seems it wasn't really necessary, I've just remove it 😄

**Review by danielwrobert — APPROVED** [review]

**Comment by danielwrobert** [comment]
I re-tested and confirmed that the 12px border radius is still in tact!

**Pull request merged**